clap_derive/src/derives/args.rs

```diff
@@ -400,10 +400,13 @@ pub fn gen_augment(
             }};
         }
 
+        let group_methods = parent_item.group_methods();
+
         quote!(
             .group(
                 clap::ArgGroup::new(#group_id)
                     .multiple(true)
```

Comment:
I'm not quite sure how to handle this. Arg groups are created implicitly, it seems like `.multiple(true)` should indeed be the default. I guess my solution will work if some explicitly provides there own `multiple` attribute, but we'll end up with two calls to `multiple`. Any suggestions?

Reply:
This is exactly what should be done and what we do with all of the other derives: put our logic first and then let the user's run to override it